modules/social_features/social_core/social_core.install

```diff
@@ -1229,3 +1229,17 @@ function social_core_update_8811() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Install the Gin theme(as admin) and Install Gin Toolbar and Coffee modules.
+ */
+function social_core_update_8812(&$sandbox) {
+  // Install Gin Admin Theme.
+  \Drupal::service('theme_installer')->install(['gin']);
+  // Set Gin Admin Theme as admin theme.
+  \Drupal::configFactory()->getEditable('system.theme')
+    ->set('admin', 'gin')
+    ->save();
+  // Install Gin Toolbar and Coffee modules.
+  \Drupal::service('module_installer')->install(['gin_toolbar', 'coffee']);
```

Comment:
Why are we adding the `coffee` module? Won't this make it much more difficult more most users?

Reply:
It was requested from Taco)